test/suites/device-onvif-camera/main_test.go

```diff
@@ -0,0 +1,82 @@
+package test
+
+import (
+	"edgex-snap-testing/test/utils"
+	"log"
+	"os"
+	"testing"
+	"time"
+)
+
+const (
+	deviceOnvifcameraSnap        = "edgex-device-onvif-camera"
+	deviceOnvifCameraApp         = "device-onvif-camera"
+	deviceOnvifcameraService     = deviceOnvifcameraSnap + "." + deviceOnvifCameraApp
+	deviceOnvifcameraServicePort = "59984"
```

Comment:
```suggestion
	deviceOnvifCameraService     = deviceOnvifCameraSnap + "." + deviceOnvifCameraApp
	deviceOnvifCameraServicePort = "59984"
```
and corresponding  changes in the usage..